src/main/java/de/cronn/diff/Main.java

```diff
@@ -26,6 +28,8 @@
 
     private static String workingDir = FileHelper.getWorkingDir();
 
+	public static Long MAX_ALLOWED_FILESIZE_DIFFERENCE = 5000000l;
```

Comment:
We should name the unit in the const name, i.e. `MAX_ALLOWED_FILESIZE_DIFFERENCE_IN_BYTES`.

Reply:
done.